Bibliotheque nationale de France.js

```diff
@@ -506,14 +541,10 @@ var BnfClass = function() {
         var url = record.getFieldSubfields("856");
 		
 		if(url && url.length > 0 && url[0].u) {		
-		 item.attachments = [
-				{
-					url: url[0].u,
-					title: "Bnf Gallica entry", 
-					mimeType: "text/html", 
-					snapshot:false
-				}
-			];
+
+		 item.url = url[0].u;
```

Comment:
It think that UNIMARC field 856 is the same as in MARC21 (please correct me if I am wrong) and there the scope is larger:

> The field may be used in a bibliographic or holdings record for a resource when it or a subset of it is available electronically. In addition, it may be used to locate and access an electronic version of a non-electronic resource described in either the bibliographic record, a portion of the resource described, or a related electronic resource.

See also the examples in http://www.loc.gov/marc/856guide.html#atta . I would rather let this as a attachment here and we can make sure that Gallica translator will handle this differently.